lib/Schema.js

```diff
@@ -87,13 +87,24 @@ function Schema(obj, options) {
 Schema.prototype.toDynamo = function(model) {
 
   var dynamoObj = {};
-  //debug('toDynamo with schema attributes', this.attributes);
-  for(var name in this.attributes) {
+
+  for(var name in model) {
+    if(!model.hasOwnProperty(name)){
+      continue;
+    }
```

Comment:
tried that and had a bunch of test failures that didn't seem worth tracking down since the solution above worked.